roles/bootstrap-os/tasks/main.yml

```diff
@@ -71,12 +71,28 @@
     - override_system_hostname
     - ansible_os_family in ['Suse', 'Flatcar Container Linux by Kinvolk', 'Container Linux by CoreOS', 'ClearLinux'] or is_fedora_coreos
 
-- name: "Install ceph-commmon package"
+- name: Install ceph-commmon package
   package:
     name:
       - ceph-common
     state: present
-  when: rbd_provisioner_enabled|default(false)
+  when:
+    - rbd_provisioner_enabled|default(false)
+    - is_fedora_coreos is false
+
+- name: Get installed rpm-ostree packages
+  raw: "export http_proxy={{ http_proxy | default('') }};rpm-ostree status | grep LayeredPackages"
+  become: true
+  register: installed_rpm_ostree_packages
+  when: is_fedora_coreos is true
+
+- name: Install ceph-common package if rbd-provisioner is enabled
+  raw: "export http_proxy={{ http_proxy | default('') }};rpm-ostree install ceph-common"
+  become: true
+  when:
+    - rbd_provisioner_enabled|default(false)
+    - is_fedora_coreos is true
+    - installed_rpm_ostree_packages.stdout.find("ceph-common") == -1
```

Comment:
Does this not require a reboot? 
I would recommend to define a fact
```
  - set_fact:
      fedora_coreos_additional_packages: "{{ (rbd_provisioner_enabled|default(false)) | ternary(['ceph-common'],[]) }}"

```
and concat this array with `fedora_coreos_packages`, so the ostree installation can be done inside `bootstrap-fedora-coreos.yml`

Reply:
I had tested it in my lab, so I did not need a reboot. But I can put one in just to be on the safe side.

if i include it in the bootstrap-fcos, is it not possible to install rbd provisioner afterwards or am i wrong?

It will check if python is installed, if not, the fcos packages will be installed, if so the packages will not be installed anymore, so will ceph-common.